### PR TITLE
5854 Fix signing requirement condition for child modules

### PR DESCRIPTION
**Pull request by crandall-chow-bl**
Simple cleanup of the signing condition, being tolerant of the task being "uploadArchives" or ":uploadArchives".

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -134,21 +134,21 @@ subprojects {
     }
 
     signing {
-        required { isReleaseVersion && gradle.taskGraph.hasTask("uploadArchives") }
+        required { isReleaseVersion && (gradle.taskGraph.hasTask('uploadArchives') || gradle.taskGraph.hasTask(':uploadArchives')) }
```
Comment (crandall-chow-bl) [inline build.gradle]: Needed for the signing requirement to be met for child module artifacts.

File: build.gradle [hunk 2]
```diff
@@ -134,21 +134,21 @@ subprojects {
...
         sign configurations.archives
     }
 
     uploadArchives {
         repositories.mavenDeployer {
             beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
```
Comment (crandall-chow-bl) [inline build.gradle]: Text formatting only.  No code changes.

**Review by rdooley — APPROVED** [review]
reasonable

**Comment by rholder** [comment]
This seems like it's fine to fix. However, I'm not 100% on how it was being signed before if it's currently not signing correctly...

**Review by rholder — APPROVED** [review]

**Pull request merged**